dev_tools/container/packages.yaml

```diff
@@ -0,0 +1,63 @@
+packages:
```

Comment:
the packages listed here are installed from the apt, right?

Reply:
yes, they are not detectable by spack (yet), so I need to specify their location manually